assets/three-dots.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><svg id="Layer_1_1_" style="enable-background:new 0 0 16 16;" version="1.1" viewBox="0 0 16 16" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><circle cx="2" cy="8" r="2"/><circle cx="8" cy="8" r="2"/><circle cx="14" cy="8" r="2"/></svg>
```

Comment:
if you want to cut size a bit you can use [SVGOMG](https://jakearchibald.github.io/svgomg/)